File: find-length-of-number/javascript-typescript/using-multiplication.ts
function findLengthUsingMultiplicationLoop() {
    let num = 123456;
    let counter = 0;
    let temp = 1;

    while (temp <= num) {
        temp *= 10;
        counter++;
    }

    console.log(`Final Length: ${counter}`)
}

findLengthUsingMultiplicationLoop();

File: find-length-of-number/c-c++/using-multiplication.c
#include <stdio.h>

int main()
{
    int num = 123456;
    int counter = 0;
    int temp = 1;

    while (temp <= num)
    {
        temp *= 10;
        counter++;
    }

    printf("%d", counter);

    return 0;
}